__tests__/integration.test.js

```diff
@@ -139,13 +139,80 @@ describe("GET: /api/articles/:article_id/comments", () => {
         expect(body).toEqual({ message: "comment not found" });
       });
   });
-    test("GET: responds with a 400 status code when passed an invalid ID", () => {
+  test("GET: responds with a 400 status code when passed an invalid ID", () => {
+    return request(app)
+      .get("/api/articles/hello/comments")
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "bad request" });
+      });
+  });
+});
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("POST: responds with a 201 status code when a new comment is added", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This was okay, wasn't super intrigued when I was reading this article",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { newComment } = body;
+
+        expect(Object.keys(newComment).length).toBe(6);
```

Comment:
I get why you've done this, and totally ok to split it up. Your next test is also asserting the data type because you are writing strings or numbers. You would be totally valid to just to the one test.